src/etc/systemd/system/atop.service.d/10-override.conf

```diff
@@ -0,0 +1,6 @@
+[Service]
+ExecStartPre=
+ExecStart=
```

Comment:
Why do we need empty and duplicate ExecStart* entries?

Reply:
To overwrite it "from the scratch".
> if one wants to remove entries from a setting that is parsed as a list (and is not a dependency), such as AssertPathExists= (or e.g. ExecStart= in service units), one needs to first clear the list before re-adding all entries except the one that is to be removed

Reply:
Looks good, thank you for the explanation